ghost/core/test/e2e-api/shared/version.test.js

```diff
@@ -129,11 +130,9 @@ describe('API Versioning', function () {
                     }]
                 });
 
-            mockManager.assert.sentEmailCount(1);
-            mockManager.assert.sentEmail({
-                subject: 'Attention required: Your Zapier integration has failed',
-                to: 'jbloggs@example.com'
-            });
+            emailMockReceiver.sentEmailCount(1);
```

Comment:
This bit actually makes a good point to stop using the `mockManager.assert` bit in favor of `emailMockReceiver.sentEmailCount`. I've pushed deprecation note into the mock manager to make it extra clear

Reply:
Cool!